jmeter-java-dsl-graphql/pom.xml

```diff
@@ -36,5 +36,17 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+    <build>
```

Comment:
We have set the source and target version of the jvm to 1.8. Any particular reason to change the graphql dependency here? If we change this, then users using 1.8 would be required to upgrade to jvm 11 (which might not be that simple in some scenarios).